besu/src/main/java/org/hyperledger/besu/services/BesuEventsImpl.java

```diff
@@ -75,4 +81,20 @@ public long addSyncStatusListener(final SyncStatusListener syncStatusListener) {
   public void removeSyncStatusListener(final long listenerIdentifier) {
     syncState.removeSyncStatusListener(listenerIdentifier);
   }
+
+  private static BlockPropagated blockPropagated(
```

Comment:
BlockPropagated may be name more like PropagatedBlockContext or something that shows that it's not a block data but meta data including network related total difficulty, hence the context word.